muse/components/trackers.py

```diff
@@ -0,0 +1,42 @@
+# flake8: noqa E501
```

Comment:
if It is not a component, can we keep this file in [utility](https://github.com/Lightning-AI/stable-diffusion-deploy/tree/main/muse/utility)?